### ISSUE TITLE
`parameters` and `contains` need somewhat more structure

### ISSUE THREAD

**Issue by keighrim** [clamsproject/mmif#160]
After #156 and #159 are merged, current (d5ff013) MMIF schema [defines](https://github.com/clamsproject/mmif/blob/d5ff0135d6c38d6858b5435f95a2570f94b5b73e/schema/mmif.json#L53-L58) `parameters` and `contains` in viewMetadata too loosely, so test cases in mmif-python SDK cannot pass for general cases. I'd like to give more structures to both objects to represent our discussion so far.

**Comment by marcverhagen** [comment]
I am in favor of tightening the schema, but why do tests fail because the schema are too permissive?

**Comment by keighrim** [comment]
Because the test framework ([hypothesis](https://hypothesis.readthedocs.io/en/latest/) and [hypothesis-jsonschema](https://pypi.org/project/hypothesis-jsonschema/)) automatically generates possible input data (constrained by the schema we provide) in the test time. Underspecified objects in the schema will allow the framework to generate samples that python SDK cannot handle.

**Comment by keighrim** [comment]
Implementation in #161 suggests that `parameters` object is a simple from strings to strings. In case of *restified* CLAMS apps (by clams-python), we can store the raw values from query strings and rest assured that the values are all string. And currently other entrypoints are out of scope of development (https://github.com/clamsproject/mmif/issues/154#issuecomment-783682252).

**Comment by keighrim** [comment]
Closed via #161 .
